src/plone/api/content.py

```diff
@@ -569,6 +569,36 @@ def get_uuid(obj=None):
     return IUUID(obj)
 
 
+@required_parameters("obj")
+def get_path(obj=None, relative_to_portal=False):
```

Comment:
```suggestion
def get_path(obj=None, relative=False):
```

That would mimic [`absolute_url`](https://github.com/zopefoundation/Zope/blob/37dcf30674e37077375268b67e91ec9e71875151/src/OFS/interfaces.py#L180)

Reply:
I like `relative_to_portal` for making it more obvious what it's relative to.